workers/unity/Assets/Playground/Scripts/UI/ConnectionScreenController.cs

```diff
@@ -64,5 +72,12 @@ private void TryConnect()
             workerConnector.ConnectionScreenController = this;
             workerConnector.TryConnect();
         }
+
+        private string GetIPFromExtras()
```

Comment:
`FromExtras` is not particularly descriptive or helpful here, would `get receptionist host ip from command line` or something be better?

Reply:
`GetReceptionistHostFromArguments` ?

Reply:
sgtm